lucene/core/src/test/org/apache/lucene/util/graph/TestGraphTokenStreamFiniteStrings.java

```diff
@@ -44,13 +48,13 @@ private void assertTokenStream(TokenStream ts, String[] terms, int[] increments)
     assertNotNull(terms);
     assertNotNull(increments);
     assertEquals(terms.length, increments.length);
-    BytesTermAttribute termAtt = ts.getAttribute(BytesTermAttribute.class);
+    CharTermAttribute termAtt = ts.getAttribute(CharTermAttribute.class);
     PositionIncrementAttribute incrAtt = ts.getAttribute(PositionIncrementAttribute.class);
     int offset = 0;
     while (ts.incrementToken()) {
       // verify term and increment
       assert offset < terms.length;
-      assertEquals(terms[offset], termAtt.getBytesRef().utf8ToString());
+      assertEquals(terms[offset], termAtt.toString());
```

Comment:
Was this change necessary, it should be equivalent right?

Reply:
It needs to change from `BytesTermAttribute` to something else - I could change it to a `TermToBytesRefAttribute`, but `CharTermAttribute` seemed simpler and involves less byte-wrangling

Reply:
ok